recipe/meta.yaml

```diff
@@ -8,27 +9,29 @@ package:
 source:
   url: https://codeload.github.com/snowflakedb/{{ name }}/tar.gz/v{{ version }}
   fn: {{ name }}-{{ version }}.tar.gz
-  sha256: 56eb57b69796ee766221d95c6b2de835c20754d6ffa5a4a35f80138355a970bb
+  sha256: {{ sha256 }}
   patches:
     - 0001-drop-pyarrow-version-numbers.patch
 
 build:
   number: 0
-  # neither on Windows 32-bit nor on s390x there is right now arrow support
-  skip: true  # [win32 or s390x or py<37 or py==310]
+  # no arrow support on win-32 or s390x
+  skip: true  # [win32 or s390x or py<37]
   script:
     - export SF_NO_COPY_ARROW_LIB=1              # [unix]
     - export SF_ARROW_LIBDIR="${PREFIX}/lib"     # [unix]
     - export ENABLE_EXT_MODULES=true             # [unix]
     - set SF_NO_COPY_ARROW_LIB=1                 # [win]
-    - set "SF_ARROW_LIBDIR=%LIBRARY_LIB%"        # [win]
+    - set "SF_ARROW_LIBDIR=%PREFIX%\Library"     # [win]
     - set ENABLE_EXT_MODULES=1                   # [win]
```

Comment:
I figured out what to change this to by trial and error. This is used to find the `.lib` libraries for use during the build process.

Reply:
I think it's LIBRARY_PREFIX not LIBRARY_LIB. I may be mistaken.

Reply:
I tried quite a few different combinations, including those. Not entirely sure why the failed.